integration-tests/docker/test_env/test_env.go

```diff
@@ -59,15 +59,23 @@ func NewTestEnv() (*CLClusterTestEnv, error) {
 	if err != nil {
 		return nil, err
 	}
-	networks := []string{network.Name}
+	n := []string{network.Name}
 	return &CLClusterTestEnv{
+		Geth:       test_env.NewGeth(n),
+		MockServer: test_env.NewMockServer(n),
 		Network:    network,
-		Geth:       test_env.NewGeth(networks),
-		MockServer: test_env.NewMockServer(networks),
 		l:          log.Logger,
 	}, nil
 }
 
+func (te *CLClusterTestEnv) WithTestEnvConfig(cfg *TestEnvConfig) *CLClusterTestEnv {
+	te.Cfg = cfg
+	n := []string{te.Network.Name}
+	te.Geth = test_env.NewGeth(n, test_env.WithContainerName(cfg.Geth.ContainerName))
+	te.MockServer = test_env.NewMockServer(n, test_env.WithContainerName(cfg.MockServer.ContainerName))
```

Comment:
Is there an option to just pass `cfg.MockServer.ContainerName` as param? `test_env.NewMockServer(n, cfg.MockServer.ContainerName`? This would make it simpler

Reply:
Ya we can do that. Would be a separate ticket that deals with CTF changes since these methods and patterns would need to be changed there.

Reply:
I see, ok!